godot_export_manager.py

```diff
@@ -318,15 +318,11 @@ def execute(self, context):
 
         path = group[self.idx].export_path
         if (path.find("//") == 0 or path.find("\\\\") == 0):
-            # If relative, convert to absolute
```

Comment:
I don't think such a comment useless, but I would have put it (maybe in shorter version) on the first line, to make the logic more explicit:
```
if (path.find("//") == 0 or path.find("\\\\") == 0): # Relative path
```